### PR TITLE
Add a section with automated installation

### PR DESCRIPTION
**Pull request by akscram**

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -271,5 +271,50 @@ the Managed Services Tooling bootstrap image.
...
+
+The command above generates an inventory file which is located at
+`inventory/inventory.cfg`. This file is used by automation playbooks.
+
+The Internate connection is needed to perform the bootstrap process which
+downloads all necessary images for further installation. Perform the bootstrap:
+
+    ::
+
+      ansible-playbook -i inventory/inventory.cfg bootstrap-runner.yml
```
Comment (corest) [inline docs/quickstart.rst]: Here should be ansible-playbook -i inventory/bootstrap.cfg bootstrap-runner.yml

File: docs/quickstart.rst [hunk 2]
```diff
@@ -271,5 +271,50 @@ the Managed Services Tooling bootstrap image.
...
+
+The deployment of OSS Tools requires at least three hardware or virtual nodes
+which are connected in the same network. These nodes have to be installed from
+the pre-built OSS Tooling ISO.
+
+.. note::
+    During the installation from ISO the `mirantis` user will be created with
+    the password `mirantis`.
+
+After installation of nodes `automation` have to be copied on one of the nodes:
```
Comment (corest) [inline docs/quickstart.rst]: Currently, we are using the separate machine for deployment. The case with deployment from the first machine was not nested

Comment (corest) [inline docs/quickstart.rst]: Currently, we are using the separate machine for deployment. The case with deployment from the first machine was not nested

File: docs/quickstart.rst (hunk 1 above)
Comment (akscram) [reply docs/quickstart.rst]: Do you mean that the separate `inventory/bootstrap.cfg` have to be used for the bootstrapping? Should I create this separate inventory somehow? But as I understand correctly the generator creates all-in-one inventory that can be used by both playbooks. Please, correct me here.

Comment (corest) [reply docs/quickstart.rst]: Nope, inventory.cfg is used for actual deployment. Bootstrap is running only on the deployment machine and doesn't require any additional configuration. This is done on idea, bootstrap and deployment can be triggered on different machines with manual moving of the deployment code

Comment (akscram) [reply docs/quickstart.rst]: In this case it also requires to move all downloaded artifacts alongside with the deployment code. Okay then let's write here that is the only tested variant is to have four nodes: one for the bootstrapping and running deployment on the rest three nodes.

**Pull request merged**